test/test_using_plams/test_package_wrapper.py

```diff
@@ -0,0 +1,59 @@
+import shutil
+from os import remove
+from os.path import isdir, isfile
+from pathlib import Path
+
+import pytest
+from scm.plams import ADFJob, AMSJob, from_smiles
+
+from qmflows import PackageWrapper, run, Settings
+from qmflows.packages.SCM import ADF_Result
+from qmflows.packages.package_wrapper import ResultWrapper
+
+PATH = Path('test') / 'test_files'
+
+
+@pytest.mark.slow
```

Comment:
You can also mock the ADF

Reply:
Would it be better to mock ADF or simply raise an exception when ADF cannot be found?
By checking for ADF environment variables for example?

Reply:
Well, the point is that for CI it is difficult to install ADF, so in general is more reliable to have a mock of the ADF result

Reply:
Fair enough, the test is aready marked as ``slow`` though, just like all other tests requiring ADF.

Reply:
If you consider that is important, you can mock the ADF result like https://github.com/SCM-NV/qmflows/blob/master/test/test_mock_operations.py

Reply:
Thanks for the info, I'll get back to you a bit later.

Reply:
See https://github.com/SCM-NV/qmflows/pull/149#discussion_r387616762.